### ISSUE TITLE
Report -- separate the stats and plots into different sections for the test set and training set.

### ISSUE THREAD

**Issue by qchiujunhao** [goeckslab/Galaxy-Ludwig#31]
Right now, some of the plots and stats are confusing—it’s unclear which dataset they are coming from.

Recommendation:
have separate tabs/sections for model performance on the train and test sets and plot confusion matrix + ROC AUC for both datasets.
